<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplayWidget</class>
 <widget class="QWidget" name="ReplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>880</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Lucida Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_leave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Leave</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prompt">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>791</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>119</y>
     <width>771</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="leftVBox_2"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>120</y>
     <width>321</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="rightVBox_2">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>840</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>840</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>›</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_last">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>840</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>»</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>840</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>‹</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_first">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>840</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>«</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_speed">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>840</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>840</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Speed:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
